### ISSUE TITLE
Change ProgressBar color?

### ISSUE THREAD

**Issue by amanzan** [Tapadoo/Alerter#83]
Is there any way to change the ProgressBar color?

**Comment by kpmmmurphy** [comment]
Not currently, but I will add it in the next release.

**Comment by kpmmmurphy** [comment]
This has been included in version 2.0.1

```java
Alerter.create(ExampleActivity.this)
                .setTitle("Alert Title")
                .setText("Alert text...")
                .enableProgress(true)
                .setProgressColorRes(R.color.colorAccent)
                .show();
```

**Comment by LuiguiBalarezo** [comment]
Hi

In version 1.0.6 there is no .enable Progress (true).

Why?.
